### PR TITLE
fix: LSDV-5333: regression where variable 'total' was not defined.

### PR DESCRIPTION
**Pull request by Axel-Jacobsen**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [x] Backend (API)
- [ ] Frontend

### Describe the reason for change

https://github.com/heartexlabs/label-studio/issues/4081

#### What does this fix?

[This commit](https://github.com/heartexlabs/label-studio/commit/1d3c9bf51f5ec6dbc8f7f29efa83745f68c96dcf) changed the name of a variable but did not update all locations of it's use in the function.

#### Does this change affect security?

no

#### What alternative approaches were there?

Reverting the commit, but then we lose the other work that the commit gave us.

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [x] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

Data importing

### CODE REVIEW

File: label_studio/data_import/uploader.py [hunk 1]
```diff
@@ -35,29 +33,34 @@ def is_binary(f):
...
 
 
 def check_max_task_number(tasks):
     # max tasks
     if len(tasks) > settings.TASKS_MAX_NUMBER:
-        raise ValidationError(f'Maximum task number is {settings.TASKS_MAX_NUMBER}, '
-                              f'current task number is {len(tasks)}')
+        raise ValidationError(
+            f'Maximum task number is {settings.TASKS_MAX_NUMBER}, '
+            f'current task number is {len(tasks)}'
```
Comment (Axel-Jacobsen) [inline label_studio/data_import/uploader.py:53]: This PR looks like a lot of changes, but you can go commit-by-commit through and see that it is really just
- fix the bug
- remove unused imports
- autoformat
- add tests

**Comment by Axel-Jacobsen** [comment]
Also, maybe [CONTRIBUTING.md](https://github.com/heartexlabs/label-studio/blob/develop/CONTRIBUTING.md) should be updated with the PR title requirements? I am just guessing by looking at other PR titles, and it wasn't in the PR description template.

File: label_studio/data_import/uploader.py (hunk 1 above)
Comment (Axel-Jacobsen) [reply label_studio/data_import/uploader.py:53]: woops wrong line! haha

File: label_studio/data_import/uploader.py [hunk 2]
```diff
@@ -35,29 +33,34 @@ def is_binary(f):
...
+        )
 
 
 def check_tasks_max_file_size(value):
     if value >= settings.TASKS_MAX_FILE_SIZE:
-        raise ValidationError(f'Maximum total size of all files is {settings.TASKS_MAX_FILE_SIZE} bytes, '
-                              f'current size is {total} bytes')
+        raise ValidationError(
+            f'Maximum total size of all files is {settings.TASKS_MAX_FILE_SIZE} bytes, '
+            f'current size is {value} bytes'
```
Comment (Axel-Jacobsen) [inline label_studio/data_import/uploader.py:61]: This PR looks like a lot of changes, but you can go commit-by-commit through and see that it is really just

- fix the bug
- remove unused imports
- autoformat
- add tests

**Comment by hogepodge** [comment]
@makseq this looks like it should be a fairly straight-forward patch to merge.

**Comment by niklub** [comment]
@Axel-Jacobsen thanks for your contribution!

**Pull request merged**